taxi/models.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self):
+        return f"{self.name} {self.country}"
+
+    class Meta:
+        verbose_name_plural = "manufacturers"
+        ordering = ("name", "country", )
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(
+        Manufacturer, related_name="cars", on_delete=models.CASCADE
+    )
+    drivers = models.ForeignKey(
+        settings.AUTH_USER_MODEL, related_name="cars", on_delete=models.CASCADE
+    )
```

Comment:
The `drivers` field in the `Car` model is currently a `ForeignKey`, which implies a one-to-many relationship. If a car can have multiple drivers, consider using a `ManyToManyField` instead to accurately represent this relationship.